packages/frontend/src/pages/Settings/SettingsPage.tsx

```diff
@@ -53,6 +56,10 @@ const SettingsPage = (): JSX.Element | null => {
               <NavItem to={`${url}/application`} description="Application" />
               <NavItem to={`${url}/period`} description="Period Defaults" />
               <NavItem to={`${url}/discord`} description="Discord Bot" />
+              <NavItem
```

Comment:
The cusom export settings page doesn't render properly here. 

"Rendered more hooks than during the previous render."